### ISSUE TITLE
Alert word handling

### ISSUE THREAD

**Issue by neiljp** [zulip/zulip-terminal#588]
We can discuss options for this, but possible ideas could include:
* [ ] Notify when an alert word is used (if narrow is not muted) [as in webapp]
* [ ] Button for alert words? Showing unread alerts? Hotkey? [not in webapp]
* [ ] Ability to configure alert words, possibly via using `i` on the button? [webapp uses settings]

**Comment by preetmishra** [comment]
@neiljp We don't markup the alert words differently (like the web app) currently, right?

I can see my alert words in 'green' in the web app but not in ZT.

**Comment by neiljp** [comment]
Agreed - added to list; we basically don't do anything with alert words right now.

**Comment by zee-bit** [comment]
This looks interesting, didn't know there was such a feature in Zulip. :sweat_smile: 
I would love to work on this. :smile:
